docs/tutorials/integrations/kserve/blue-green.md

```diff
@@ -0,0 +1,262 @@
+
+![Blue-Green rollout](images/blue-green.png)
+
+???+ "Before you begin"
+    1. Ensure that you have the [kubectl CLI](https://kubernetes.io/docs/reference/kubectl/).
+    2. Have access to a cluster running [KServe](https://kserve.github.io/website). You can create a [KServe Quickstart](https://kserve.github.io/website/0.10/get_started/#before-you-begin) environment as follows:
+    ```shell
+    curl -s "https://raw.githubusercontent.com/kserve/kserve/release-0.10/hack/quick_install.sh" | bash
+    ```
+<!-- Istio 1.15 already installed 
+    3. Install [Istio](https://istio.io). You can install the [demo profile](https://istio.io/latest/docs/setup/getting-started/).
+    ```shell
+    istioctl install --set profile=demo -y
+    ```
+-->
+
+## Install the Iter8 controller
+
+--8<-- "docs/tutorials/installiter8controller.md"
+
+## Deploy a primary model
+
+Deploy the primary version of a model using an `InferenceService`:
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: wisdom-0
+  labels:
+    app.kubernetes.io/name: wisdom
+    app.kubernetes.io/version: v0
+    iter8.tools/watch: "true"
+spec:
+  predictor:
+    minReplicas: 1
+    model:
+      modelFormat:
+        name: sklearn
+      runtime: kserve-mlserver
+      storageUri: "gs://seldon-models/sklearn/mms/lr_model"
+EOF
+```
+
+??? note "About the primary `InferenceService`"
+    Naming the model with the suffix `-0` (and the candidate with the suffix `-1`) simplifies the rollout initialization. However, any name can be specified.
+    
+    The label `iter8.tools/watch: "true"` lets Iter8 know that it should pay attention to changes to this `InferenceService`.
+
+Inspect the deployed `InferenceService`:
+
+```shell
+kubectl get inferenceservice wisdom-0
+```
+
+When the `READY` field becomes `True`, the model is fully deployed.
+    
+## Initialize the Blue-Green routing policy
+
+Initialize model rollout with a blue-green traffic pattern as follows:
+
+```shell
+#cat <<EOF | helm template traffic --repo https://iter8-tools.github.io/iter8 traffic-templates -f - | kubectl apply -f -
+cat <<EOF | helm template /Users/kalantar/projects/go.workspace/src/github.com/iter8-tools/iter8/charts/traffic-templates -f - | kubectl apply -f -
+templateName: initialize-rollout
+targetEnv: kserve
+trafficStrategy: blue-green
+modelName: wisdom
+EOF
+```
+
+The `initialize-rollout` template (with `trafficStrategy: blue-green`) configures the Istio service mesh to route all requests to the primary version of the model (`wisdom-0`). Further, it defines the routing policy that will be used by Iter8 when it observes changes in the models. By default, this routing policy splits inference requests 50-50 between the primary and candidate versions. For detailed configuration options, see the Helm chart.
```

Comment:
At the end "see the Helm chart". It would be nice to add a hyperlink.

Reply:
done